docs/developer/app-store/apps/taxes.mdx

```diff
@@ -0,0 +1,97 @@
+
+`NODE_ENV` (_optional_)
+
+Controls the environment in which the app is running.
+
+The possible values are: `development`, `production`.
+
+`PORT` (_optional_)
+
+The port on which the app will run. `3000` by default.
+
+`ALLOWED_DOMAIN_PATTERN` (_optional_)
+
+A regex pattern that prohibits the app from being installed on a Saleor instance that does not match the pattern. If not set, all installations will be allowed.
+
+## Configuration
+
+### Channels
+
+For a channel to appear on the _Channels_ list, you must first select **"use tax app"** as its tax calculation method. You can do that on the _Tax Configuration_ page (_Configuration_ → _Taxes_).
+
+In the _Taxes_ page, select the channel and change the value of _Select the method of tax calculation_ to "_Use tax app_". Then, click _Save_ and come back to the Taxes App.
+
+The channel should now appear on the _Channels_ list. You can now fill in the "ship from" address and assign a tax provider instance to the channel (once you create it in the next step).
+
+### Providers
+
+In the _Providers_ tab, you can create a new tax provider instance. Each tax provider instance is a different set of credentials for a specific tax provider.
+
+To create a new instance, click the tax provider button to see the provider configuration form.
+
+Here are the required fields for each provider:
+
+#### TaxJar
+
+- Instance name - a name for the instance. It will be used to identify the instance across all the views.
+- API Key - [the API key](https://developers.taxjar.com/api/reference/#authentication) for the TaxJar account.
+- Sandbox - a flag that indicates whether the instance should use [the sandbox environment or the production environment](https://developers.taxjar.com/api/reference/#sandbox-environment).
+
+#### Avatax
+
+- Instance name - a name for the instance. It will be used to identify the instance across all the views.
+- Sandbox - a flag that indicates whether the instance should use [the sandbox environment or the production environment](https://developer.avalara.com/erp-integration-guide/sales-tax-badge/authentication-in-avatax/sandbox-vs-production/).
+- Autocommit - a flag that indicates whether the instance should [automatically commit the transactions](https://developer.avalara.com/communications/dev-guide_rest_v2/commit-uncommit/).
+- Username - the username for the Avatax account. You will generate it in the Avalara dashboard (_Settings_ → _License and API keys_ → _License key_).
+- Password - the password for the Avatax account. You will generate it in the Avalara dashboard (_Settings_ → _License and API keys_ → _License key_).
+- Company code - the [company code](https://developer.avalara.com/ecommerce-integration-guide/sales-tax-badge/transactions/simple-transactions/company-codes/) for the Avatax account.
```

Comment:
I think this part *can* be in docs, something like "configuring providers", but I'd remove part where you explain to click a button